feat: Add April's fact (#21)
octopusthink/funfactsaboutoctopus.es: src/content/facts/12-de-montfort.md

---
title: Giant octopuses really aren't a thing.
source: Wikipedia
sourceUrl: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
image: /images/colossal-octopus.jpg
imageSource: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
---

> Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. And you know how, when you think about something a whole lot, you start seeing it everywhere? Yep, that's what happened here.

Reviewer: On first pass this sentence read a bit awkwardly to me. Here's my suggestion to tighten it up a little:

```suggestion
Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. You know how thinking about something a lot, you start seeing it everywhere? Yup, that's what happened here.
```

> de Montfort told everyone they'd sunk a group of warships that mysteriously disappeared one night in 1782. Turns out the disappearance was less of a mystery than originally proposed and he was summarily discredited. His career tanked (probably attacked by a giant octopus), and he died destitute. 

Reviewer: "They'd" is _quite_ an ambiguous noun here—I thought it meant that de Montfort told people that the people had sunk a warship. How about we clarify:

```suggestion
One night in 1782, when a group of warships mysteriously disappeared, de Montfort claimed it was the work of giant octopuses. Turns out the disappearances weren't a mystery, but were a secret kept by the British Navy. de Montfort was discredited, his career tanked (probably attacked by a giant octopus), and he died destitute.
```
